ml4h20.bib

```diff
@@ -2,20 +2,20 @@ @Proceedings{ML4H-2019
     booktitle = {Proceedings of the Machine Learning for Health NeurIPS Workshop},
     name = {Machine Learning for Health Workshop},
     shortname = {ML4H},
-    editor = {Adrian V. Dalca and Matthew Mcdermott and Emily Alsentzer and Sam Finlayson and Michael Oberst and Fabian Falck and Brett Beaulieu-Jones},
+    editor = {Adrian V. Dalca and Matthew B.A. McDermott and Emily Alsentzer and Samuel G. Finlayson and Michael Oberst and Fabian Falck and Brett Beaulieu-Jones},
```

Comment:
These changes need to be made in the file that is linked to from the `[edit]` link in the main proceedings page. i.e., this file: https://github.com/mlresearch/v116/edit/gh-pages/_config.yml 

Changing the .bib file will not update the proceedings.